### PR TITLE
Add CVC Field

### PR DESCRIPTION
**Pull request by dmytrokhl**
## Description of changes
Add cvc field with cvc icon

### CODE REVIEW

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+#if os(iOS)
+import UIKit
+#endif
+
+/// An object that displays an editable text area. Can be use instead of a `VGSTextField` when need to detect and show credit card brand images.
+public final class VGSCVCTextField: VGSTextField {
+  
+    internal let cvcIconView = UIImageView()
+    internal lazy var stackView = self.makeStackView()
+    internal let stackSpacing: CGFloat = 8.0
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift:18]: Do we have fixed spacing?

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        return stack
+    }
+  
+    // override textFieldDidChange
+    override func textFieldValueChanged() {
+        super.textFieldValueChanged()
+        updateCardImage()
+    }
+  
+    func updateCardImage() {
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift:113]: Mark with `internal`.

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+       if let state = state as? CardState {
+          cvcIconView.image = (cardsIconSource == nil) ? state.cardBrand.brandIcon :  cardsIconSource?(state.cardBrand)
+       } else {
+        cvcIconView.image = VGSPaymentCards.unknown.brandIcon
+       }
+    }
+  
+    func setCVCIconAtLocation(_ location: CVCIconLocation) {
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift:121]: Mark with `internal` non public.

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+        switch location {
+        case .left:
+            stackView.insertArrangedSubview(cvcIconView, at: 0)
+        case .right:
+            stackView.addArrangedSubview(cvcIconView)
+        }
+    }
+    
+    func updatecvcIconViewSize() {
+        if let widthConstraint = cvcIconView.constraints.filter({ $0.identifier == "widthConstraint" }).first {
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift:132]: Mark with `internal`.

Comment (dmytrokhl) [reply Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift:120]: No need in internal extension.

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift (hunk 3 above)
Comment (dmytrokhl) [reply Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift:109]: No need in internal extension.

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift (hunk 2 above)
Comment (dmytrokhl) [reply Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift]: No need in internal extension.

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+      let heightConstraint = NSLayoutConstraint(item: cvcIconImageView,
+                                                attribute: .height,
+                                                relatedBy: .equal,
+                                                toItem: nil,
+                                                attribute: .notAnAttribute,
+                                                multiplier: 1,
+                                                constant: cvcIconSize.height)
+      heightConstraint.identifier = "heightConstraint"
+      // fix conflict with textfield height constraint when card icon more higher than textfield
+      heightConstraint.priority = .required
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSCVCTextField.swift:174]: Remove lines 173, 174 and PR can be merged.

**Review by EugeneIOs — APPROVED** [review]

**Pull request merged**